docs/deploy.rst

```diff
@@ -243,3 +243,67 @@ After everything starts, which may take a few minutes, you should be able
 to visit your Girder instance at ``http://X.X.X.X`` where ``X.X.X.X`` is the
 IP address in the container description above. Congratulations, you
 have a full Girder instance available on Google Container Engine!
+
+Elastic Beanstalk
+-----------------
+
+Girder comes with pre-packaged configurations for deploying onto Elastic Beanstalk's
+`Python platform <http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/concepts.platforms.html#concepts.platforms.python>`_
+(both 2.7 and 3.4).
+
+The configurations live within ``devops/beanstalk`` and are designed to be copied into your working Girder directory
+at deploy time.
+
+The following assumes you have a a checked out copy of Girder (using git) and an existing MongoDB instance which
+can be accessed by your Beanstalk application.
+
+From within the checked out copy of Girder, install and configure the CLI tools: ::
+
+  $ pip install awscli awsebcli
+  $ aws configure
+
+Initialize the Beanstalk application with a custom name. This is an interactive process
+that will ask various questions about your setup: ::
+
+  $ eb init my-beanstalk-app
+
+Build Girder and its client-side assets locally: ::
+
+  $ pip install .[plugins]  # optionally only build specific plugins
+  $ girder-install web  # optionally specific plugins with --plugins
+
+.. note:: Since Girder is unable to restart and load plugins in the Beanstalk environment,
+	  plugins may be enabled/disabled but will require a restart of Beanstalk application
+	  servers to take effect.
+
+Create a requirements.txt for the Beanstalk application: ::
+
+  $ pip freeze | grep -v '^girder\|^awscli\|^awsebcli' > requirements.txt
```

Comment:
should you mention working in a virtualenv created for this deployment, since you are pip freezing?